flag/service/operator/operator.go

```diff
@@ -0,0 +1,7 @@
+package operator
+
+type Operator struct {
+	FrameworkBackOffDuration string
```

Comment:
I saw that joe was using these flags when reviewing his code a few days back.

Reply:
When you think it is useful go with it. I do not see it being useful. /shrug